android/docs/guides/installation.md

```diff
@@ -0,0 +1,91 @@
+## Getting Started
+
+Until the app is available in the
+[Play Store](https://play.google.com/store?hl=en_US), it requires manual
+installation from a host machine. Due to restrictions in dataset redistribution,
+some amount of manual retrieval, processing and packaging is required to use
+certain task datasets for accuracy evaluation. We're actively investigating how
+we can bundle the necessary datasets directly in the app.
+
+If you're primarily interested in testing inference latency, skip directly to
+the [App Execution](#app-execution) section. The app can run MLPerf latency
+measurements without the need for full dataset installation.
+
+### Dataset installation
+
+A tasks-specific dataset is required required for measuring accuracy.
+
+This currently requires some manual work to prepare and distribute the relevant
+datasets. Note that the full datasets can be extremely large; a subset of the
+full dataset can be prepared to give a useful proxy for task-specific accuracy.
+When prepared, each task directory should be pushed to
+`/sdcard/mlperf_datasets/...`.
+
+#### Imagenet (Classification)
+
+The app evaluates Image Classification via MobileNet V1 (float & quantized),
+using any subset of images used in the
+[ILSVRC 2012 image classification task](http://www.image-net.org/download-images/).
+
+To download the 2012 dataset, you will need to do the following:
+
+*   Open the
+    [ImageNet downloads link](http://www.image-net.org/download-images),
+    creating an ImageNet account if you have not done so already.
+*   Navigate to the 2012 downloads page under the **`Download links to ILSVRC
+    image data`** section.
+*   Download the **`Validation images (all tasks)`** archive.
+*   Create a staging directory for the dataset, call it
+    `${WORKING_DIR}/imagenet`.
+*   From the downloaded archive, extract the images into a
+    `${WORKING_DIR}/imagenet/img` directory.
+
+The full set of validation images is quite large, over 6GB. This can take an
+extremely long time to process, so it is *highly* recommended that you use a
+smaller subset of images. We recommend leaving ~300 images in
```

Comment:
If you want a smaller set of images just use the images from performance mode. The point of the full datasets is to validate accuracy. If you don't use the full dataset, what is the point?